### ISSUE TITLE
Incorrect otherwise warning

### ISSUE THREAD

**Issue by ndmitchell** [ndmitchell/hlint#517]
```
otherwise = True
```

Gives:

```
Sample.hs:2:1: Warning: Used otherwise as a pattern
Found:
  otherwise
Perhaps:
  _
```

**Comment by ndmitchell** [comment]
Seems to have been fixed.
